src/app/view/components/dot-portlet-base/components/dot-portlet-toolbar/dot-portlet-toolbar.component.html

```diff
@@ -2,13 +2,13 @@
     <div class="p-toolbar-group-left" data-testId="leftGroup">
         <h3 data-testId="title" *ngIf="title">{{ title }}</h3>
         <div class="dot-portlet-toolbar__extra-left" data-testId="leftExtra">
-            <ng-content select="#left"></ng-content>
+            <ng-content select="[left]"></ng-content>
```

Comment:
Why?

Also, does this have test? can we add it? I completely miss check if this can have test.

Reply:
test added. regarding the change it feels more clean to have an `attribute` than a `id`